citizenvoice/apiapp/urls.py

```diff
@@ -0,0 +1,25 @@
+# ====================================================================================================================
+#
```

Comment:
I checked that the api root works, but I not the paths to questions and such. Maybe you can add some examples of how to use the API in the next pull request. 

![Screenshot from 2022-08-29 11-42-16](https://user-images.githubusercontent.com/8195424/187173949-ce763553-47ca-4f73-a710-88af0555d874.png)
![Screenshot from 2022-08-29 11-43-15](https://user-images.githubusercontent.com/8195424/187173954-cfb5d515-47fc-4632-9be8-c78678b09371.png)